### DISCUSSION TITLE
2D cross-Hole

### DISCUSSION THREAD

**Discussion by pouryaalidoust** [gimli-org/pyGIMLi#453] [General]
Hi,

I am following the 2D cross hole example(https://www.pygimli.org/_examples_auto/seismics/plot_3_crosshole_tomography.html) in your website with a little bit of difference in the geometry. I have two regions with velocities of 200 and 1000 but noticed that the inverted models value is not exceeding 300 no matter what I use as the starting model.

![image](https://user-images.githubusercontent.com/75387608/198047165-32c6e7e0-8246-4a23-8572-d448cd394078.png)

Even in the model explained in your website, the difference between the max and min inverted velocity is 300 (when I increased the difference between the circular anomalies and the surrounding soil, the result was not acceptable in terms of the velocity value!
My file has been attached. Would be grateful on any tips how to improve the results?
Here is the link to file:

https://drive.google.com/file/d/1kfOb7383QtiSSr54FuYteqvbWyr7frF9/view?usp=sharing

**Comment by halbmy** [comment]
I am very sorry. While regular issues are usually answered, discussions disappear from the screen, which is why I just found this one  by accident. The point in your example is the noise level of 8% (`noiseLevel=0.08`) which results in an apparent velocity image (`tt.showData(data)`) of:
![image](https://github.com/gimli-org/gimli/assets/7136401/e2a48c33-cd58-41f1-8028-35d5d14037fe)

So one cannot expect to image the body. In traveltime the absolute noise level `noiseAbs` should dominate. A choice of `relativeNoise=0` results in 
![image](https://github.com/gimli-org/gimli/assets/7136401/d7c9297c-0edd-487c-873b-ce28c7863fe7)

however the contrast in apparent velocity is much smaller. With some good options, the inversion results can be much better
![image](https://github.com/gimli-org/gimli/assets/7136401/824b60ae-a649-4caa-ab78-56d7334f121a)

at least in regions of good coverage. I put the whole notebook at
https://github.com/gimli-org/notebooks/blob/main/issues/453/issue453.ipynb
